### PR TITLE
[45] Product Page Using HTML

### PR DESCRIPTION
**Pull request by ayotola-t**
## Task: `Product Page for eCommerce Store`
 Changes I made to given task include:

- Cleaning up the images folder in my html submissions folder
- Renaming the images
- Creating a new branch and pushing from it
- Including the footer in the body of my code

### CODE REVIEW

**Review by adaugochi — COMMENTED** [review]
Remove spaces on your image names. Also, this task requires that you make use of two images only. Please delete the other two

File: submissions/ayotolaTehingbola/html/product.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            <ul>
+                <li> <a href="https://www.Twitter.com">Twitter</a></li>
+                <li> <a href="https://www.Instagram.com">Instagram</a></li>
+                <li> <a href="https://www.meduim.com">Meduim</a></li>
+            </ul>
+            <p> &copy; Ayotola 2020. All rights reserved. </p>
+        </footer>  
+
+    </body>
+
```
Comment (adaugochi) [inline submissions/ayotolaTehingbola/html/product.html:64]: Remove too many new line spaces

**Comment by ayotola-t** [comment]
> Remove spaces on your image names. Also, this task requires that you make use of two images only. Please delete the other two

Used only two images
Screenshot 2020-09-28 at 5 22 24 PM. 


Screenshot 2020-09-28 at 5 22 16 PM

**Review by adaugochi — CHANGES_REQUESTED** [review]
Look for a better descriptive name for `bagg`

**Review by adaugochi — APPROVED** [review]

**Pull request merged**